gremlin-go/driver/graphTraversal.go

```diff
@@ -587,7 +587,8 @@ func (g *GraphTraversal) TimeLimit(args ...interface{}) *GraphTraversal {
 
 // Times adds the times step to the GraphTraversal.
 func (g *GraphTraversal) Times(args ...interface{}) *GraphTraversal {
-	g.Bytecode.AddStep("times", args...)
+	// Gremlin server only accepts times step with integer argument values
+	g.Bytecode.AddStep("times", int32Args(args)...)
```

Comment:
Current cucumber test suite still uses int32() cast for the Times arguments (example [here](https://github.com/lyndonbauto/tinkerpop/blob/7ada477e466479d5378d72a476df6dd2b8a78941/gremlin-go/driver/cucumber/gremlin.go#L63)), the `GolangTranslator.java` will need to be updated at this [line](https://github.com/lyndonbauto/tinkerpop/blob/7ada477e466479d5378d72a476df6dd2b8a78941/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/translator/GolangTranslator.java#L276) and `gremlin.go` file regenerated to reflect this change.

Reply:
I don't think this matters too much. Passing in a int32 cast is still a valid input, and the cucumber tests are meant to test input-output, not the semantics of how it's entered into gremlin-go. There are other examples of inputs not being exactly as you'd expect a real user to pass things in.